.github/workflows/build_fiberplane_hono_playground.yml

```diff
@@ -0,0 +1,82 @@
+name: Build @fiberplane/playground and @fiberplane/hono
+
+on:
+  pull_request:
+    branches: ["*"]
+    paths:
+      - "packages/fiberplane-hono/**"
+      - "packages/playground/**"
+  push:
+    branches: ["main", "release-*"]
+    paths:
+      - "packages/fiberplane-hono/**"
+      - "packages/playground/**"
+
+env:
+  CARGO_TERM_COLOR: always
+  FORCE_COLOR: true
+
+jobs:
+  build_packages:
+    name: Build packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          run_install: false
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+          cache: "pnpm"
+
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: 1.9.4
+
+      - name: Install dependencies
+        run: pnpm install
+        env:
+          CI: true
+
+      # Linting: we use global biome command
+      # any extra commands should be added to the lint:ci script
+      # in the package.json
+
+      - name: Lint all workspaces (biome)
+        run: biome ci .
+
+      - name: Lint all workspaces (eslint or other linters)
+        run: pnpm --recursive lint:ci
+
+      - name: Typecheck all workspaces
```

Comment:
Maybe rename this to something more specific? Perhaps something like:
`typecheck playground & hono packages`